lisa/tests/base.py

```diff
@@ -1301,8 +1302,13 @@ def run_rtapp(cls, target, res_dir, profile=None, ftrace_coll=None, cg_cfg=None,
         profile = profile or cls.get_rtapp_profile(target.plat_info)
         cg_cfg = cg_cfg or cls.get_cgroup_configuration(target.plat_info)
 
+        trace_events = [event.replace('rtapp_', '')
```

Comment:
Should that be turned into a helper somewhere?

Reply:
I cannot really see any other location that would need to know these are userspace rtapp events. The only reason we need to know that here is to be able to enable them in rtapp JSON when somebody tries to collect them.